apps/api/src/app/feature-flags/services/launch-darkly.service.ts

```diff
@@ -39,4 +37,18 @@ export class LaunchDarklyService implements IFeatureFlagsService {
       }
     }
   }
+
+  public async initialize(): Promise<void> {
+    try {
+      await this.client.waitForInitialization();
+      Logger.log('Launch Darkly SDK has been successfully initialized', LOG_CONTEXT);
+    } catch (error) {
+      Logger.error('Launch Darkly SDK has failed when initialized', LOG_CONTEXT, error);
+      throw error;
+    }
+  }
+
+  private mapToLaunchDarklyContext(context: IFeatureFlagContext): LDContext {
+    return context as LDContext;
```

Comment:
I'm getting here the TS error:
![Screenshot 2023-05-18 at 22 20 09](https://github.com/novuhq/novu/assets/2607232/b344e5e6-3baa-4366-acc8-e5dc555374ae)

Reply:
btw do you plan to have a logic here, because this casting doesn't feel right?

Reply:
This bit is meant to be touched in this task: https://linear.app/novu/issue/NV-2324/mappers-around-launch-darkly-api
But I am unsure how the Launch Darkly context is going to be the best way, therefore this useless function.
I am fixing the build error though. Thanks for raising it.